### ISSUE TITLE
Is there any plan to provide pretrained weights of Mip-NeRF 360 that can replicate the paper results?

### ISSUE THREAD

**Issue by yzslab** [google-research/multinerf#10]
Hi,
Seems like it need to take a lot of time to replicate the paper results if training from scratch.
It will be better if you can provide some pretrained weights.
Thanks.

**Comment by jonbarron** [comment]
Do you mean the optimized NeRF parameters for each individual scene? No, we have no plan to release those models.

Note that NeRF models do not have a traditional "training" step --- when NeRF is applied to a set of images, its weights are randomly initialized and then optimized with those images using gradient descent. There is no "pretraining".

**Comment by yzslab** [comment]
> Do you mean the optimized NeRF parameters for each individual scene? No, we have no plan to release those models.
> 
> Note that NeRF models do not have a traditional "training" step --- when NeRF is applied to a set of images, its weights are randomly initialized and then optimized with those images using gradient descent. There is no "pretraining".

OK, thanks for your reply.
